themes/default/layouts/shortcodes/pulumi-command.html

```diff
@@ -1,5 +1,12 @@
 {{/* Fetch the current version of the Pulumi CLI doc." suffix. */}}
-{{- $json := getJSON "https://api.github.com/repos/pulumi/docs/contents/content/docs/reference/cli/pulumi.md" }}
+{{ $headers := dict }}
+{{ $gitHubToken := os.Getenv "GITHUB_TOKEN" }}
+{{ if ne $gitHubToken "" }}
+    {{ $token := printf "Bearer %s" $gitHubToken }}
```

Comment:
optional suggestion: You can shorten this with `with` if you'd like:

```suggestion
{{ with (os.Getenv "GITHUB_TOKEN") }}
    {{ $token := printf "Bearer %s" . }}
```

Empty string evaluates to a non-truthy value in Go's template system.